src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -3141,6 +3141,30 @@ public static void DCS_KnownTypeMethodName()
         Assert.Equal(value.emps[1].Name, actual.emps[1].Name);
     }
 
+    [Fact]
+    public static void DCS_NativeDll()
+    {
+        NativeDllWrapper.CallIntoNativeDll();
+    }
+
+    [Fact]
+    public static void DCS_ReadOnlyDictionaryCausingDuplicateInvalidDataContract()
```

Comment:
This test failed on netfx as test `DCS_ReadOnlyDictionary` did because of #18312.

@stone-li can you please move the test next to `DCS_ReadOnlyDictionary` and put the following attribute on it?

`[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #18312")]`